### PR TITLE
Prompt when env not supplied

### PR DESCRIPTION
**Pull request by nsarupr**

### CODE REVIEW

File: opta/layer.py [hunk 1]
```diff
@@ -220,14 +221,22 @@ def load_from_dict(cls, conf: Dict[Any, Any], env: Optional[str]) -> Layer:
                     )
                 potential_envs[env_name] = (current_parent, env_meta)
 
-            if len(potential_envs) > 1 and env not in potential_envs:
+            if env is None:
+                if len(potential_envs) == 1:
+                    env = list(potential_envs.keys())[0]
+                else:
+                    """This is a repeatable prompt, which will not disappear until a valid choice is provided or SIGABRT
```
Comment (juandiegopalomino) [inline opta/layer.py:228]: you tested this out?

Comment (nsarupr) [reply opta/layer.py:228]: The test cases with only one Environment passed.
I'll be adding a few more test cases to it.

Comment (juandiegopalomino) [reply opta/layer.py:228]: what I meant did you run this locally, and verified that it keeps prompting you until a valid answer is given?

Comment (nsarupr) [reply opta/layer.py:228]: Yes, it keeps prompting until you enter a valid answer.

Comment (nsarupr) [reply opta/layer.py:228]: Also, I have removed the default value to accidentally avoid a default value.

**Review by juandiegopalomino — APPROVED** [review]

**Pull request merged**